src/components/Parcels.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Image from "next/image";
+import type {Parcel} from "@/utils/parcel-fetch"
+
+
+
```

Comment:
This line should be removed to follow the standard of having 2 empty lines between imports and code

Reply:
This looks good now